cmd/openshift-install/gather.go

```diff
@@ -132,38 +137,53 @@ func runGatherBootstrapCmd(directory string) (string, error) {
 		}
 
 		if bootstrap == "" || len(masters) == 0 {
-			return "", errors.New("bootstrap host address and at least one control plane host address must be provided")
+			return "", "", errors.New("bootstrap host address and at least one control plane host address must be provided")
 		}
 	} else if bootstrap == "" || len(masters) == 0 {
-		return "", errors.New("must provide both bootstrap host address and at least one control plane host address when providing one")
+		return "", "", errors.New("must provide both bootstrap host address and at least one control plane host address when providing one")
 	}
 
 	return gatherBootstrap(bootstrap, port, masters, directory)
 }
 
-func gatherBootstrap(bootstrap string, port int, masters []string, directory string) (string, error) {
+func gatherBootstrap(bootstrap string, port int, masters []string, directory string) (string, string, error) {
+	gatherID := time.Now().Format("20060102150405")
+
+	serialLogBundle := filepath.Join(directory, fmt.Sprintf("serial-log-bundle-%s.tar.gz", gatherID))
+	serialLogBundlePath, err := filepath.Abs(serialLogBundle)
+
+	logrus.Info("Pulling platform specific console logs from the bootstrap machine")
+	gather, err := gather.New(logrus.StandardLogger(), serialLogBundle, bootstrap, masters, directory)
+	if err != nil {
+		return "", "", errors.Wrap(err, "Failed to create gather object")
```

Comment:
Should be best effort: don't stop here if we fail, keep trying to get the rest of the logs.

Reply:
Fixed.